docs/home/whats-new-2-16.md

```diff
@@ -66,20 +48,22 @@ Fast, reliable, and easy-to-use unified interface.
 
 #### Authentication and Authorization
 
-* <!--  DONE RUN-13107/RUN-13108 - SSO users visibility-->Added a column to the *Users* table for the type of user that was created (Local or SSO). You can now also view the user's access role assignments directly from the *Users* table by pressing *VIEW*. In addition, SSO users who have logged into the system will now be visible in the *Users* table. For more information, see [Adding, Updating, and Deleting Users](../admin/admin-ui-setup/admin-ui-users.md#adding-updating-and-deleting-users).
+* <!--  DONE RUN-13107/RUN-13108 - SSO users visibility-->Added a column to the *Users* table for the type of user that was created (Local or SSO). You can now also view the user's access role assignments directly from the *Users* table by pressing *View*. In addition, SSO users who have logged into the system will now be visible in the *Users* table. For more information, see [Adding, Updating, and Deleting Users](../admin/admin-ui-setup/admin-ui-users.md#adding-updating-and-deleting-users).
```

Comment:
```suggestion
* <!--  DONE RUN-13107/RUN-13108 - SSO users visibility-->SSO users who have logged into the system will now be visible in the *Users* table. In addition, added a column to the *Users* table for the type of user that was created (Local or SSO). For more information, see [Adding, Updating, and Deleting Users](../admin/admin-ui-setup/admin-ui-users.md#adding-updating-and-deleting-users).
```

I removed the part about viewing the access rules of the user (it already existed before this version). Also changed the order from the major thing to the less important one..